src/term/check/mod.rs

```diff
@@ -1,10 +1,14 @@
-use super::{DefId, DefinitionBook, Name, Term};
+use super::{DefId, DefNames, DefinitionBook, Name, Term};
 
 pub mod unbound_vars;
 
 impl DefinitionBook {
   pub fn check_has_main(&self) -> anyhow::Result<DefId> {
-    if let Some(main) = self.def_names.def_id(&Name::new("main")) {
+    if let Some(main) = self
```

Comment:
We should only allow one of the accepted entry point names. Otherwise it becomes confusing for the user in case they for some reason define the two names and expect the other one to be the entrypoint